packages/www/pages/dashboard/streams/[id]/health.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useCallback } from "react";
+import { Box, Flex, Link, Text } from "@livepeer.com/design-system";
+import Layout from "../../../../layouts/dashboard";
+import { useRouter } from "next/router";
+import { useApi } from "../../../../hooks";
+import { useEffect, useState } from "react";
+
+import Chart from "@components/Chart";
+import VideoContainer from "@components/TestPlayer/videoContainer";
```

Comment:
I'd like to avoid using the old theme-ui components outside the `Dashboard` directory. Can we use the `Player` component inside the `Dashboard` directory instead? This is the one I use on the stream detail view. The plan is to phase out `theme-ui`.